.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,55 @@
+name: run-tests
+
+on: [push, pull_request]
+
+jobs:
+  run-tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-18.04
+          - ubuntu-latest
+        vim-flavour:
+          - vim
+          - neovim
+        vim-from-testing:
+          - true
+          - false
+        exclude:
+          - os: ubuntu-18.04
+            vim-from-testing: true
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Add hirsute repos
+        if: ${{ matrix.os == 'ubuntu-latest' && matrix.vim-from-testing }}
+        run: sudo apt-add-repository 'deb http://archive.ubuntu.com/ubuntu/ impish main universe'
+
+      - name: Install vim
+        if: ${{ matrix.vim-flavour == 'vim' }}
+        run: |
+          sudo apt update
+          sudo apt install vim vim-gtk xvfb
+      - name: Install neovim
+        if: ${{ matrix.vim-flavour == 'neovim' }}
+        run: |
+          sudo apt update
+          sudo apt install neovim python3-neovim xvfb
+      - name: Install vroom
+        run: |
+          wget https://github.com/google/vroom/releases/download/v0.14.0/vroom_0.14.0-1_all.deb
```

Comment:
Nit: Could we put these versions in an env var, like in https://github.com/google/vim-codefmt/blob/793d816524934e6553c76437120eea5df8e85a1e/.travis.yml#L22?

Reply:
Done.